tests/unit/test_faultydata.py

```diff
@@ -30,63 +29,47 @@ def getFullPath(self, localId):
 class TestVariantSetNoIndexedVcf(FaultyVariantDataTest):
     localIds = ["no_indexed_vcf"]
 
-    @unittest.skip("Skipping until DB based repo refactor is done")
     def testInstantiation(self):
         for localId in self.localIds:
             path = self.getFullPath(localId)
-            self.assertRaises(
-                exceptions.NotIndexedException,
-                variants.HtslibVariantSet, self.dataset, localId, path,
-                None)
+            variantSet = variants.HtslibVariantSet(self.dataset, localId)
+            with self.assertRaises(exceptions.NotIndexedException):
+                variantSet.populateFromDirectory(path)
 
 
 class TestInconsistentMetaData(FaultyVariantDataTest):
     localIds = ["inconsist_meta"]
 
-    @unittest.skip("Skipping until DB based repo refactor is done")
     def testInstantiation(self):
         for localId in self.localIds:
             path = self.getFullPath(localId)
-            variantSet = variants.HtslibVariantSet(
-                self.dataset, localId, path, None)
+            variantSet = variants.HtslibVariantSet(self.dataset, localId)
+            variantSet.populateFromDirectory(path)
```

Comment:
The exception is thrown whenever `checkConsistency` is called.  Should we move the `checkConsistency` calls to be run whenever data files are added?